server/src/js/components/map/outlook.js

```diff
@@ -3,16 +3,13 @@
 // It uses the MapContainer
 // TODO: needs refactoring into layers and styles
 // ALSO need to fix the functionality, I don't think the tickets have been developed as of 31/01/2020
```

Comment:
Think we can remove these comments now too

Reply:
Comments removed.